osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -366,13 +380,18 @@ public virtual bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
             if (!this.IsCurrentScreen()) return false;
 
+            if (game == null)
+                return false;
+
+            var flattenedMods = ModUtils.FlattenMods(game.AvailableMods.Value.SelectMany(kv => kv.Value));
```

Comment:
This breaks the mod hotkey tests (`OsuGame` not present in DI hierarchy, previously `OsuGameBase` was used which was present).

Reply:
As per PR opening post, this an RFC and I expected tests to break. I made it because I know I'm not the only one with strong opinions on code quality that potentially mean throwing out the whole PR (seems like that may happen).

Reply:
The intention was only to explain why tests broke, nothing more.